### PR TITLE
translation brazilian portuguese

### PR DESCRIPTION
**Pull request by alvarorichard**
Hello, I would like to add the Brazilian Portuguese translation to the README.md. I translated it as faithfully and comprehensibly as possible to the original. I only did the translation, without adding anything else besides that. I kindly ask you to consider this contribution and review it for approval.
@stevhliu and @MKhalusova

### CODE REVIEW

File: README_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,581 @@
...
+- [Respostas de Perguntas Visuais com ViLT](https://huggingface.co/dandelin/vilt-b32-finetuned-vqa)
+- [Classificação de Imagens sem Anotação com CLIP](https://huggingface.co/openai/clip-vit-large-patch14)
+- [Respostas de Perguntas em Documentos com LayoutLM](https://huggingface.co/impira/layoutlm-document-qa)
+- [Classificação de Vídeo sem Anotação com X-CLIP](https://huggingface.co/docs/transformers/model_doc/xclip)
+Em Tarefas Multimodais:
+- [Respostas de Perguntas em Tabelas com TAPAS](https://huggingface.co/google/tapas-base-finetuned-wtq)
+- [Respostas de Perguntas Visuais com ViLT](https://huggingface.co/dandelin/vilt-b32-finetuned-vqa)
+- [Classificação de Imagens sem Anotação com CLIP](https://huggingface.co/openai/clip-vit-large-patch14)
+- [Respostas de Perguntas em Documentos com LayoutLM](https://huggingface.co/impira/layoutlm-document-qa)
+- [Classificação de Vídeo sem Anotação com X-CLIP](https://huggingface.co/docs/transformers/model_doc/xclip)
```
Comment (stevhliu) [inline README_pt-br.md:128]: This looks like a duplicate of the above ;)

```suggestion
```

File: README_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,581 @@
...
+>>> classifier = pipeline("sentiment-analysis")
+
+# Classifique o texto como positivo ou negativo
+>>> classifier("Estamos muito felizes em apresentar o pipeline no repositório dos transformers.")
+[{'label': 'POSITIVE', 'score': 0.9996980428695679}]
+```
+
+A segunda linha de código baixa e armazena em cache o modelo pré-treinado usado pelo pipeline, enquanto a terceira linha o avalia no texto fornecido. Neste exemplo, a resposta é "positiva" com uma confiança de 99,97%.
+
+Muitas tarefas têm um pipeline pré-treinado pronto para uso, não apenas em PNL, mas também em visão computacional e processamento de áudio. Por exemplo, podemos facilmente extrair objetos detectados em uma imagem:
```
Comment (stevhliu) [inline README_pt-br.md:162]:
```suggestion
Muitas tarefas têm um `pipeline` pré-treinado pronto para uso, não apenas em PNL, mas também em visão computacional e processamento de áudio. Por exemplo, podemos facilmente extrair objetos detectados em uma imagem:
```

File: README_pt-br.md [hunk 3]
```diff
@@ -0,0 +1,581 @@
...
+Consulte a [página de instalação do TensorFlow](https://www.tensorflow.org/install/), a [página de instalação do PyTorch](https://pytorch.org/get-started/locally/#start-locally) e/ou [Flax](https://github.com/google/flax#quick-install) e [Jax](https://github.com/google/jax#installation) páginas de instalação para obter o comando de instalação específico para a sua plataforma.
+
+Quando um desses back-ends estiver instalado, o 🤗 Transformers pode ser instalado usando pip da seguinte forma:
+
+```bash
+pip install transformers
+```
+Se você deseja experimentar com os exemplos ou precisa da versão mais recente do código e não pode esperar por um novo lançamento, você deve instalar a [biblioteca a partir do código-fonte](https://huggingface.co/docs/transformers/installation#installing-from-source).
+
+## Com conda
```
Comment (stevhliu) [inline README_pt-br.md:282]:
```suggestion
### Com conda
```

File: README_pt-br.md [hunk 4]
```diff
@@ -0,0 +1,581 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
```
Comment (stevhliu) [inline README_pt-br.md:2]:
```suggestion
Copyright 2023 The HuggingFace Team. All rights reserved.
```

**Review by stevhliu — COMMENTED** [review]
Thanks for your translation! :)

**Comment by alvarorichard** [comment]
I have already applied the suggested changes

**Review by stevhliu — APPROVED** [review]
Thanks for your translation!

**Pull request merged**